client/webpack.client.base.config.js

```diff
@@ -59,7 +60,7 @@ module.exports = {
     loaders: [
       { test: /\.(woff2?|svg)$/, loader: 'url?limit=10000' },
       { test: /\.(ttf|eot)$/, loader: 'file' },
-      { test: /\.(jpe?g|png|gif|svg|ico)$/, loader: 'url?limit=10000' },
+      { test: /\.(jpe?g|png|gif|svg|ico)$/, loader: `customfile-loader?limit=10000&name=[hash]_[name].[ext]${assetHost}` },
```

Comment:
@rap1ds, I'm not sure how (consistently) browsers handle "image@2x.jpg" cases if the name is in format "image@2x-39487593563874594.jpg"?